package.json

```diff
@@ -13,6 +13,7 @@
     "bunTest": "bun test --coverage --bail"
   },
   "dependencies": {
+    "@nextui-org/card": "^2.0.28",
```

Comment:
No need to install @nextui-org/card. The @nextui-org/react already contain the card elements. That will cause duplication in dependencies.